hw/bsp/stm32h7/boards/stm32h750bdk/board.mk

```diff
@@ -0,0 +1,21 @@
+# STM32H745I-DISCO uses OTG_FS
+# FIXME: Reset enumerates, un/replug USB plug does not enumerate
+
+CFLAGS += -DSTM32H750xx -DCORE_CM7 -DHSE_VALUE=25000000
+
+# Default is FulSpeed port
+PORT ?= 0
+
+# GCC
+SRC_S_GCC += $(ST_CMSIS)/Source/Templates/gcc/startup_stm32h745xx.s
```

Comment:
startup and linker should be changed to h750 instead of h745. I was about to change that but I realize that H750 mcu only has 128KB of flash, it proabably means to be used with qspi flash. Though if all examples can fit into 128KB, we are all good.

Reply:
Missed that... I have just found that there is no 750 linker script :/